### PR TITLE
Add Get-GcpsSubscription

### PR DESCRIPTION
**Pull request by quoctruong**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/397)

### CODE REVIEW

File: Google.PowerShell/PubSub/GcpsSubscription.cs [hunk 1]
```diff
@@ -153,4 +153,166 @@ protected override void ProcessRecord()
...
+
+        /// <summary>
+        /// <para type="description">
+        /// The names of the subscriptions to be retrieved.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false, Position = 0)]
+        [ValidateNotNullOrEmpty]
+        [Alias("Name")]
+        public string[] Subscription { get; set; }
```
Comment (iantalarico) [inline Google.PowerShell/PubSub/GcpsSubscription.cs]: I think I asked this before but this isn't plural due to convention correct?

File: Google.PowerShell/PubSub/GcpsSubscription.cs [hunk 2]
```diff
@@ -153,4 +153,166 @@ protected override void ProcessRecord()
...
+                        WriteObject(response.Subscriptions, true);
+                    }
+                    listRequest.PageToken = response.NextPageToken;
+                }
+                while (!Stopping && listRequest.PageToken != null);
+            }
+        }
+
+        /// <summary>
+        /// Given a list of subscription names, returns the corresponding subscriptions.
```
Comment (iantalarico) [inline Google.PowerShell/PubSub/GcpsSubscription.cs]: " returns the corresponding subscriptions." Should this be "writes the corresponding subscriptions."?

File: Google.PowerShell/PubSub/GcpsSubscription.cs (hunk 1 above)
Comment (quoctruong) [inline Google.PowerShell/PubSub/GcpsSubscription.cs]: Yes, this is due to convention https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx

File: Google.PowerShell/PubSub/GcpsSubscription.cs (hunk 2 above)
Comment (quoctruong) [inline Google.PowerShell/PubSub/GcpsSubscription.cs]: Fixed.

**Comment by quoctruong** [comment]
Thanks!

**Pull request merged**